tests/e2e_test.go

```diff
@@ -42,5 +42,5 @@ var _ = ginkgo.BeforeSuite(func() {
 var _ = ginkgo.AfterSuite(testUtils.TearDownNetwork)
 
 var _ = ginkgo.Describe("[Teleporter integration tests]", func() {
-	ginkgo.It("Send a message from Subnet A to Subnet B", BasicOneWaySend)
+	ginkgo.It("Send a message from Subnet A to Subnet B", BasicOneWaySendGinkgo)
```

Comment:
Instead of separate `BasicOneWaySendGinkgo` function, can we call `BasicOneWaySend(&network.LocalNetwork{})` here?

Reply:
No, we can't provide any arguments to the function. Ginkgo only supports functions with no parameters or return values in this interface.

Reply:
can we rename to `BasicOneWaySendLocal` or something similar?

Reply:
I'd propose getting rid of the `XXXGinkgo()` methods all together, and instead update these lines to be along the line of:
```
ginkgo.It("Send a message from Subnet A to Subnet B", func() { flows.BasicOneWaySend(&network.LocalNetwork{}) })
```

I think that will likely be the clearest for new developers looking at the code for the first time, compared to possible confusion of why the `XXXGinkgo()` methods are needed or should be used currently.

Reply:
fixed.